source/nodejs/adaptivecards-designer/src/containers/theme-supported-host-container.ts

```diff
@@ -0,0 +1,53 @@
+import * as Adaptive from "adaptivecards";
+import { HostContainer } from "./host-container";
+
+export enum ColorTheme {
+    Light = "Light",
+    Dark = "Dark"
+}
+
+export abstract class ThemeSupportedHostContainer extends HostContainer {
+    private _colorTheme: ColorTheme;
```

Comment:
According to https://www.typescriptlang.org/static/TypeScript%20Classes-83cc6f8e42ba2002d5e2c04221fa78f9.png

![image](https://user-images.githubusercontent.com/6290692/192631718-75410852-1b8d-4984-b282-7d432e55685e.png)

It's nice to use `#field` for private.
and possible 
```
private _colorTheme: ColorTheme = ColorTheme.Light;;
```

Reply:
we may stay with `private ` other than `#` because our code base is widely use private

Reply:
Okay sounds good :)